apps/civicsignalblog/src/payload/blocks/WebTools/PageHeader.js

```diff
@@ -0,0 +1,30 @@
+import { slateEditor } from "@payloadcms/richtext-slate";
+
+import richText from "#civicsignalblog/payload/fields/richText";
+
+const PageHeader = {
+  slug: "webtools-page-header",
+  labels: {
+    singular: "Page Header",
+    plural: "Page Headers",
+  },
+  imageURL: "/images/cms/blocks/web_tools_page_header.png",
+  imageAltText: "Header for web tools pages",
+  fields: [
+    {
+      name: "title",
+      required: true,
+      type: "text",
+    },
+    richText({
```

Comment:
If `webtools` and `research` both have `title` and `description` for a page header, can't we make both work with the same block(s0?

Reply:
The only difference between the two is that on `research` the description is a normal text field but in `webtools` we are using `RichText` as the description

Reply:
I noticed that:

1. Do we actually have a page header in [`research`](https://research.civicsignal.africa/)?
2. I was asking if it wouldn't make sense to make the description in research also supports `richtext` (since they're all part of the same app family)

Reply:
We currently don't use it anywhere on `research` front-end, but looking at how the page header would be rendered  it might not be the best option to support `richtext` since its already pre formatted 👇 
![Screenshot 2024-10-08 at 15 08 45](https://github.com/user-attachments/assets/da774c5e-f485-49c2-9b6a-9d2b92fbcdd4)
 
---
One Important question, on the `MediaData`, should we still  go with the dynamic rendering of blocks i.e. Payload fully controls what is being rendered on the front end or should we revert to the static structure and only parse whatever is coming from payload to the structure we already have

Reply:
... I'm assuming it's "**This is a subtitle**"? I think lets just give uses control over it. 👇🏽 still looks good.
<img width="964" alt="Screenshot 2024-10-08 at 15 24 55" src="https://github.com/user-attachments/assets/6b34c5d6-3237-404a-99a0-e5a06bb5741f">

compared to

<img width="964" alt="Screenshot 2024-10-08 at 15 26 08" src="https://github.com/user-attachments/assets/f30c471f-546c-4cbd-8d09-ebef8754c5d7">

Reply:
Yes that's the subtitlbe, let me add the support for richtext as well 👍